web/backend/src/Server.ts

```diff
@@ -224,13 +245,12 @@ app.post('/api/add_role', (req, res) => {
 
 // This call (only for admins) allow an admin to remove a role to a user.
 app.post('/api/remove_role', (req, res) => {
-  if (!isAuthorized(['admin'], req)) {
+  if (!isAuthorized(req.session.userid, 'database', 'removerole')) {
```

Comment:
something I forgot to mention, you might want to create string constants for the `'election'`, `'create'` and the other enforcement parameters. This way, it'll be harder for someone to make a typo 😁